### PR TITLE
make metallb config consistent again

### PR DESCRIPTION
**Pull request by benmountjoy111**

### CODE REVIEW

**Review by Vandiver247 — APPROVED** [review]

File: manifests/metallb-config.yaml [hunk 1]
```diff
@@ -1,50 +1,26 @@
...
       namespace: metallb
-  values:
-    ipAddressPool: "###ZARF_VAR_METALLB_IP_ADDRESS_POOL###"
-    interface: "###ZARF_VAR_METALLB_INTERFACE###"
+      name: metallb-config
+  timeout: 5m
+  postBuild:
+    substitute:
+      ZARF_REGISTRY: "###ZARF_REGISTRY###"
+      ZARF_REGISTRY_AUTH_PULL: "###ZARF_REGISTRY_AUTH_PULL###"
```
Comment (mikevanhemert) [inline manifests/metallb-config.yaml:23]: I think these are holdovers from a very old version of Zarf and can be removed

**Review by mikevanhemert — APPROVED** [review]
One happy-to-glad but looks solid otherwise!

File: manifests/metallb-config.yaml (hunk 1 above)
Comment (benmountjoy111) [reply manifests/metallb-config.yaml:23]: ok, you want me to remove all the other ones too? they are present in all the kustomization manifests

Comment (mikevanhemert) [reply manifests/metallb-config.yaml:23]: Maybe we can do that later as a cleanup action?

**Pull request merged**